runtime/oti/VMHelpers.hpp

```diff
@@ -1593,10 +1593,12 @@ class VM_VMHelpers
 
 #if JAVA_SPEC_VERSION >= 16
 	/**
-	 * @brief Converts the type of the return value to the return type intended for JEP389/419 FFI downcall/upcall
+	 * @brief Converts the type of the return value to the return type intended for JEP389/419 FFI downcall
+	 *
 	 * @param currentThread[in] The pointer to the current J9VMThread
 	 * @param returnType[in] The type of the return value
 	 * @param returnStorage[in] The pointer to the return value
+	 * @param isDownCall[in] A flag indicating whether the current invocation is returned from the downcall
```

Comment:
Why was this added (I don't see a parameter named `isDownCall`)?

Reply:
https://github.com/eclipse-openj9/openj9/pull/15747